### PR TITLE
Python project boilerplate

### PR DESCRIPTION
**Pull request by larsbutler**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+import setuptools
+
+
+VERSION = '0.1'
+
+setuptools.setup(
+    name='chessboard',
+    version=VERSION,
+    maintainer='RAX???',
```
Comment (larsbutler) [inline setup.py]: What should the `maintainer` and `maintainer_email` be?

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
+"""Chessboard
+
+Model application topologies and deploy applications in a
+provider-agnostic way.
+"""
+
+import setuptools
+
+
+VERSION = '0.1'
```
Comment (ziadsawalha) [inline setup.py]: 0.1.0 for the first, experimental release of a package per https://docs.python.org/2/distutils/setupscript.html

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    maintainer_email='<public mailing list>?',
+    url='https://github.com/checkmate/chessboard',
+    description=('Model application topologies and deploy application in a '
+                 'provider-agnostic way'),
+    platforms=['any'],
+    packages=setuptools.find_packages(
+        exclude=['chessboard.tests', 'chessboard.tests.*']
+    ),
+    provides=['chessboard (%s)' % VERSION],
+    license='BSD',
```
Comment (ziadsawalha) [inline setup.py]: Apache 2.0

File: setup.py (hunk 1 above)
Comment (ziadsawalha) [inline setup.py]: "Rackspace Hosting, Inc.” for maintainer. Not sure about the email. If it is required, we need to make sure it is accessible from the public and monitored.

File: setup.py (hunk 3 above)
Comment (larsbutler) [inline setup.py]: Oops, copy/paste error. I'll fix that.

File: setup.py [hunk 4]
```diff
@@ -0,0 +1,37 @@
+"""Chessboard
```
Comment (ziadsawalha) [inline setup.py]: `.` at end - PEP-257

File: setup.py [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+    url='https://github.com/checkmate/chessboard',
+    description=('Model application topologies and deploy application in a '
+                 'provider-agnostic way'),
+    platforms=['any'],
+    packages=setuptools.find_packages(
+        exclude=['chessboard.tests', 'chessboard.tests.*']
+    ),
+    provides=['chessboard (%s)' % VERSION],
+    license='Apache License 2.0',
+    keywords='wkb wkt geojson',
```
Comment (ziadsawalha) [inline setup.py]: Maybe start with something useful like `application model topology deployment manage orchestration configuration automation checkmate` - that last one to link it to the suite of apps.

Comment (larsbutler) [inline setup.py]: Oh, yeah. Another copy/paste mistake from another library I wrote. :)

**Pull request merged**